Userland/Libraries/LibJS/Bytecode/Op.h

```diff
@@ -49,6 +49,21 @@ class LoadImmediate final : public Instruction {
     Value m_value;
 };
 
+class LoadString final : public Instruction {
```

Comment:
This already exists! It's called `NewString` :^)

Reply:
Oops!